tests/functional/cypress/e2e/form-design-advancedfield.cy.js

```diff
@@ -0,0 +1,307 @@
+import 'cypress-keycloak-commands';
+import 'cypress-drag-drop';
+import { formsettings } from '../support/login.js';
+
+const depEnv = Cypress.env('depEnv');
+
+
+Cypress.Commands.add('waitForLoad', () => {
+  const loaderTimeout = 60000;
+
+  cy.get('.nprogress-busy', { timeout: loaderTimeout }).should('not.exist');
+});
+describe('Form Designer', () => {
+
+    beforeEach(()=>{
+      
+      
+      cy.on('uncaught:exception', (err, runnable) => {
+        // Form.io throws an uncaught exception for missing projectid
+        // Cypress catches it as undefined: undefined so we can't get the text
+        console.log(err);
+        return false;
+      });
+    });
+    it('Visits the form settings page', () => {
+    
+    
+        cy.viewport(1000, 1100);
+        cy.waitForLoad();
+        formsettings();
+        
+    }); 
+    it('Checks the simpleradioadvanced', () => {
+
+    cy.viewport(1000, 1800);
+    cy.waitForLoad();
+    
+ // Form design page with advanced Fields components
+    cy.get('button').contains('Advanced Fields').click();
+    cy.get('button').contains('Advanced Fields').click();
```

Comment:
Is there a specific reason why `Advanced Fields` gets clicked twice?

Reply:
@jasonchung1871 It works only if clicks twice